v2/scripts/deploy/readme.md

```
@@ -84,6 +84,115 @@ forge script scripts/deploy/deterministic/DeployGatewayZEVM.s.sol \
+  --private-key <PRIVATE_KEY> \
+  --rpc-url <RPC_URL> \
+  --verify \
+  --verifier blockscout \
+  --verifier-url <VERIFIER_URL> \
+  --chain-id <CHAIN_ID> \
+  --broadcast
+```
+
+After the implementation is deployed, the contract can be upgraded by calling the following function from the admin:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)